### ISSUE TITLE
Can not connect to streaming node

### ISSUE THREAD

**Issue by JamesShaw777** [tmcphee/cyberghostvpn#12]
### Describe the issue
A clear description of what this issue is.
can not connect to streaming node,
### Log / Screenshots
[ Please redact any personal information such as IPs, Emails, and Passwords ]
Place log outputs or screenshots here. So I can see where the issue occurred. 
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1
net.ipv6.conf.eth0.disable_ipv6 = 1
net.ipv4.ip_forward = 1
Firewall stopped and disabled on system startup
Verifying Login Auth...
Passed
Allowing PORT 53 - IN/OUT
Prepare Wireguard connection ...
Select server ... seattle-s404-i24
Connecting ... 
VPN connection established.
Blocking PORT 53 - IN/OUT
***********CyberGhost Connection Info***********
        IP: 
        CITY: 
        REGION: 
        COUNTRY: 
        DNS: 99.83.181.72

**************User Defined Variables**************
        ACC: [PASSED - NOT SHOWN]
        PASS: [PASSED - NOT SHOWN]
        COUNTRY: US
        ARGS: --streaming 'Netflix US'

 Dazn:                                  No
 HotStar:                               No
 Disney+:                               No
 Netflix:                               Originals Only
 YouTube Premium:                       Yes
 Amazon Prime Video:                    Yes (Region: US)
 TVBAnywhere+:                          Yes
 iQyi Oversea Region:                   US

### Troubleshooting
Have you tried the troubleshooting steps in the [README](https://github.com/tmcphee/cyberghostvpn#Troubleshooting)?
Was any other steps taken to resolve the issue? Please describe

**Comment by JamesShaw777** [comment]
Already done,the streaming unblock depends dns.

**Comment by tmcphee** [comment]
You can set a custom DNS with the NAMESERVER variable. Where NAMESERVER=x.x.x.x
https://github.com/tmcphee/cyberghostvpn#custom-dns--nameserver

CyberGhost has a list of DNS you can use https://my.cyberghostvpn.com/en_US/smart-dns
